components/Filters.vue

```diff
@@ -1,28 +1,51 @@
 <template>
-  <select class="bg-white dark:bg-slate shadow rounded w-full" v-model="region">
+  <!-- <select class="bg-white dark:bg-slate shadow rounded w-full" v-model="region">
     <option selected>Filter By Region</option>
     <option v-for="option in regions" :key="option" :value="option">
       {{ option }}
     </option>
-  </select>
+  </select> -->
```

Comment:
I see what you did here. It was also my first approach before I converted to a select element for semantics and accessibility reasons. While it's functional it still doesn't do the work of actually filtering the UI based on region. I think because there's no reactivity between the filtered countries computed properties list and this filter component.